.travis.yml

```diff
@@ -25,6 +25,7 @@ addons:
       - libaio-dev
       - libprotobuf-c0-dev
       - libprotobuf-dev
+      - socat
```

Comment:
is this an internal dep or a testing dependency?

Reply:
This is also needed by portforward https://github.com/containerd/cri-containerd/blob/master/pkg/server/sandbox_portforward.go#L62.

Portforward basically forward a pod port to a remote port. https://kubernetes.io/docs/tasks/access-application-cluster/port-forward-access-application-cluster/

Reply:
Ok, no wonder the CNI code adds 1-2 seconds on create time for most CRI implementations.

Reply:
go 1.10 was released, should we remove this now? :innocent:

Reply:
We can get rid of nsenter very soon! :)
But for socat, we may need some more code change.

Reply:
@mlaventure Hm, `socat` will be even easier. :)

I think we can get rid of nsenter and socat soon after we switch to golang 1.10

Reply:
Updated https://github.com/containerd/cri-containerd/issues/144#issuecomment-366387938.